docs/source/tutorials/stable_diffusion.mdx

```diff
@@ -356,11 +356,33 @@ To avoid Neuron device out of memory, it's suggested to finish all base inferenc
 
 Latent Consistency Models (LCMs) were proposed in [Latent Consistency Models: Synthesizing High-Resolution Images with Few-Step Inference by Simian Luo, Yiqin Tan, Longbo Huang, Jian Li, and Hang Zhao](https://huggingface.co/papers/2310.04378). LCMs enable inference with fewer steps on any pre-trained LDMs, including Stable Diffusion and SDXL.
 
-Here is an example to compile a LCM of Stable Diffusion XL: [latent-consistency/lcm-sdxl](https://huggingface.co/latent-consistency/lcm-sdxl) and run inference on AWS Inferentia 2 :
+In `Optimum-Neuron`, you can use the class `NeuronLatentConsistencyModelPipeline` to compile and run inference of LCMs distilled from Stable Diffusion (SD) models, and continue to use the class `NeuronStableDiffusionXLPipeline` for LCMs distilled from SDXL models.
+
+Here are examples to compile the LCMs of Stable Diffusion ( [SimianLuo/LCM_Dreamshaper_v7](https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7) ) and Stable Diffusion XL( [latent-consistency/lcm-sdxl](https://huggingface.co/latent-consistency/lcm-sdxl) ), and then run inference on AWS Inferentia 2 :
 
 ### Compile LCM
 
-***Export via Python API***
+***LCM of Stable Diffusion***
+
+```python
+from optimum.neuron import NeuronLatentConsistencyModelPipeline
+
+model_id = "SimianLuo/LCM_Dreamshaper_v7"
+num_images_per_prompt = 1
+input_shapes = {"batch_size": 1, "height": 768, "width": 768, "num_images_per_prompt": num_images_per_prompt}
+compiler_args = {"auto_cast": "matmul", "auto_cast_type": "bf16"}
```

Comment:
I would format those in multiple lines as I found it easier to read in the doc.

Reply:
Yeah, but as we put all features of sd under the same section, I try to let the snippet not look too long...

So far with the scroll bar I find it ok to read so preferably keep it that way.